packages/patternfly-4/react-docs/src/components/propsTable/propsTable.js

```diff
@@ -54,7 +54,7 @@ export const PropsTable = ({ name, description: preface, props, enumValues }) =>
             </TD>
             <TD align="center">{prop.required && <ExclamationCircleIcon />}</TD>
             <TD>{Boolean(prop.defaultValue) && prop.defaultValue.value}</TD>
-            <TD>{prop.description && prop.description.text}</TD>
+            <TD>{prop.description && <span dangerouslySetInnerHTML={{__html: prop.description.text}}/>}</TD>
```

Comment:
You're going to have to sanitize this prop if you want to use dangerouslySetInnerHTML.  There's no protection from injecting JavaScript, which can lead to cross site scripting.

Reply:
Can you use something like this to make sure the prop is okay https://www.npmjs.com/package/dompurify

Reply:
Sure @dlabaj I can look into that, however, this is only for our internal api documentation; it is not for public consumable code/components.  Do you think dompurify is necessary in this case?  -thanks

Reply:
Okay after further discussion since it's only on the doc side it should be okay.  Thanks @dtaylor113 .